modules/minimum-required-permissions-ipi-gcp-provided-sas.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="minimum-required-permissions-ipi-gcp-provided-sas_{context}"]
+= Required GCP permissions for user-provided service accounts
+
+When you are installing	a cluster, the compute and control plane nodes require their own service accounts.
+The default scenario occurs when the user does not provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
+In this case the installation program will create a service account for the control plane and compute nodes, requiring additional permissions.
+For more information, see the _Installation configuration parameters for GCP_ page.
+
+[IMPORTANT]
+====
+The following roles are required for user-provided service accounts for control plane and compute nodes respectively.
+====
+
+.Required roles	for control plane nodes
+[%collapsible]
+====
+* `roles/compute.instanceAdmin`
+* `roles/compute.networkAdmin`
+* `roles/compute.securityAdmin`
```

Comment:
During my 4.17 testing of installing into GCP shared VPC, the control-plane service account doesn't have the role `roles/compute.securityAdmin`, see [comment](https://issues.redhat.com/browse/OCPBUGS-38689?focusedId=25637781&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25637781) section "[2]". cc @barbacbd @patrickdillon

Reply:
Got it, the role is required in case of non-shared-VPC installation. Thanks!